api/app/services/models_service.py

```
@@ -0,0 +1,190 @@
+            except Exception as e:
+                logger.error(f"Error loading metadata from {metadata_file}: {e}")
+        return {}
+
+    def _load_config(self, path: Path) -> Optional[dict[str, Any]]:
+        """Load model config."""
+        config_file = path / "config.json"
+        if config_file.exists():
+            try:
+                with open(config_file) as f:
+                    return json.load(f)
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)